tasks/run-binary-builder-integration-specs/run.sh

```diff
@@ -19,7 +19,7 @@ pushd binary-builder
   bundle install --jobs="$(nproc)"
 
   if [ "${RUN_ORACLE_PHP_TESTS-false}" = "true" ]; then
-    apt-get update && apt-get -y install awscli
+    sudo zypper refresh && sudo zypper install -y aws-cli
```

Comment:
`zypper install` will automatically refresh, unless you say `zypper --no-refresh install`.